src/plugins/intel_cpu/src/nodes/kernels/scaled_attn/executor_pa.cpp

```diff
@@ -2538,7 +2538,7 @@ struct MHAHelper {
                             score_block_ptr,
                             reinterpret_cast<DATA_TYPE*>(score),
                             1,
-                            rnd_up(cur_kv_len, _block_size),
+                            cur_kv_len,
```

Comment:
NOTE for the change: The `score_output` is aligned to 64 bytes(aka 16 floats), `rnd_up(cur_kv_len, _block_size)` will align to 32 floats then 2 threads may write to same address and make random wrong result. `cvt_add` already handles tails so the padding may be removed.